searchlib/src/tests/stackdumpiterator/stackdumpiteratortest.cpp

```diff
@@ -244,6 +244,7 @@ StackDumpIteratorTest::RunTest(int testno, bool verify)
         stack.Push(new search::ParseItem(search::ParseItem::ITEM_NUMTERM, "foo", "[0;22]"));
         stack.Push(new search::ParseItem(search::ParseItem::ITEM_PREFIXTERM, "bar", "baz"));
         stack.Push(new search::ParseItem(search::ParseItem::ITEM_PHRASE, 3, "bar"));
+        stack.Push(new search::ParseItem(search::ParseItem::ITEM_SAME_ELEMENT, 3, "bar"));
```

Comment:
there should be no view here, this seems different from what I would expect the qrs to send us...